gu-hub/src/server.rs

```diff
@@ -70,41 +73,38 @@ impl config::HasSectionId for HubConfig {
 }
 
 pub struct ServerModule {
-    active: bool,
+    daemon_command: DaemonCommand,
     config_path: Option<String>,
 }
 
 impl ServerModule {
     pub fn new() -> Self {
         ServerModule {
-            active: false,
+            daemon_command: DaemonCommand::None,
             config_path: None,
         }
     }
 }
 
 impl Module for ServerModule {
     fn args_declare<'a, 'b>(&self, app: App<'a, 'b>) -> App<'a, 'b> {
-        app.subcommand(SubCommand::with_name("server").about("Hub server management"))
+        app.subcommand(DaemonHandler::subcommand())
```

Comment:
This seems too esoteric at a glance. A future reader of this code will not get it at once.
We should allow defining subcommand `about` and ideally `name`.